Gemfile

```diff
@@ -1,6 +1,11 @@
 source 'https://rubygems.org'
 
+gem 'gmaps4rails'
 
+gem 'bootstrap', '~> 4.0.0.beta2.1'
```

Comment:
@BrendinGreen versioning seems interesting, are we using a beta version?

Reply:
Thats the one they recommend (bootstrap 4 has been out for quite a while), however if we want to go back to 3 it will have no major changes.